ehitchhiking-ui/src/app/shared/guards/login.guard.ts

```diff
@@ -0,0 +1,16 @@
+import {UserService} from '../services/user.service';
+import {CanActivate, ActivatedRouteSnapshot, RouterStateSnapshot, Router} from '@angular/router';
+import {Observable} from 'rxjs';
+
+export class LoginGuard implements CanActivate {
+	constructor(private userService: UserService, private router: Router) {}
+
+	canActivate(route: ActivatedRouteSnapshot, state: RouterStateSnapshot): Observable<boolean> | boolean {
```

Comment:
Why Observable<boolean> is required? You always return true

Reply:
such default realization of canActive method. Always return boolean or Observable<boolean>